Update release ticket with communication section (#6712)
medic/cht-core: .github/ISSUE_TEMPLATE/z_release_major.md

- [ ] Create a release in GitHub from the release branch so it shows up under the [Releases tab](https://github.com/medic/cht-core/releases) with the naming convention `<major>.<minor>.<patch>`. This will create the git tag automatically. Link to the release notes in the description of the release.
- [ ] Confirm the release build completes successfully and the new release is available on the [market](https://staging.dev.medicmobile.org/builds/releases). Make sure that the document has new entry with `id: medic:medic:<major>.<minor>.<patch>`
- [ ] Upgrade the `demo-cht.dev` instance to this version.
- [ ] Follow the instructions for [releasing other products](https://docs.communityhealthtoolkit.org/core/guides/releasing/) that have been updated in this project (eg: medic-conf, medic-gateway, medic-android).
- [ ] Add the release to the [Supported versions](https://docs.communityhealthtoolkit.org/core/overview/supported-software/) and update the EOL date and status of previous releases.

> # Communicating

Reviewer: The other headings are in chronological order (although it's not clear that's the case). Putting the draft blog post down here will likely mean it's delayed by a week or two after the release actually happens. Why do you think it's helpful to move these steps down in the order?

Author: These items (except for the announcements) can be done in parallel to others in the release checklist. They may need some polishing towards the end of the release to make sure they are accurate, but mostly complete.

> # Communicating

Author: ```suggestion
# Communicating - Product Manager
```

> # Communicating

Author: ```suggestion
# Communicating - Product Manager
```

- [ ] Create a Google Doc in the [blog posts folder](https://drive.google.com/drive/u/0/folders/0B2PTUNZFwxEvMHRWNTBjY2ZHNHc) with the draft of a blog post promoting the release based on the release notes above. Once it's ready ask Max and Kelly to review it.

> - [ ] Create release artwork

Reviewer: The rest of these steps are designed to be performed by the release manager (usually an engineer). This step is usually done by Nicole. Would it make sense to say "Ask Nicole to create release artwork"? And again this is a step that can be started earlier so it's ready to go once the product is released.

Author: yup updated

> - [ ] Create release artwork

Author: ```suggestion
- [ ] Ask Nicole to create release artwork
```

Following our support policy, versions {{versions}} are no longer supported. Projects running these versions should start planning to upgrade in the near future. For more details read our software support documentation: https://docs.communityhealthtoolkit.org/core/overview/supported-software/

See what's scheduled for the next releases: https://github.com/orgs/medic/projects?query=is%3Aopen+sort%3Aname-asc
```
- [ ] Announce the release on the [CHT forum](https://forum.communityhealthtoolkit.org/c/product/releases/26), under the "Product - Releases" category. You can use the previous message and omit `@channel`.

> - [ ] Announce the release of new documenation on the [CHT Forum](https://forum.communityhealthtoolkit.org/c/product/documentation/28), under the "Product - Documentation" category.

Reviewer: Can you give an example of what this should look like, so the release manager has something to work from? How will the release manager find "new documentation"? Shouldn't the new documentation be released and announced incrementally rather than all in one go when the product is released, or is this just for documentation of new features in the release?

Author: Updated verbage to reflect documentation that pertains to features and improvements in the release. This is a new part of the process so no example to point to yet. Once I do one, I will add a template.

> - [ ] Announce the release of new documenation on the [CHT Forum](https://forum.communityhealthtoolkit.org/c/product/documentation/28), under the "Product - Documentation" category.

Author: ```suggestion
- [ ] Announce the release of new documentation for features and improvements in the release on the [CHT Forum](https://forum.communityhealthtoolkit.org/c/product/documentation/28), under the "Product - Documentation" category.
```

> - [ ] Schedule a Release communication call to educate stakeholders on product and documentation improvements

Reviewer: Who should do this step?

Author: All of the steps in the "Communicating" section are for product management. Should this be clarified?

Reviewer: Yeah that would be good. Maybe each heading should indicate who is to lead it?

Author: add "Product Manager" to the "Communicating" section. I'm less familiar with the other sections so will leave to you to update as makes sense.
